src/web.cpp

```diff
@@ -320,6 +322,7 @@ void setup_web()
         HeapSelectIram ephemeral;
 #endif
         server.on("/update", HTTP_POST, handle_update, handle_firmware_upload);
+        server.on("/verify", HTTP_POST, handle_verify, handle_firmware_verify);
```

Comment:
@dkerr64 is there a way to combine these into 1?

Reply:
btw... we can probably also pass in the firmware size and MD5 in the same query string... replacing the need to first call `setgdo()` with those values.